code-challenges.test.js

```diff
@@ -1,5 +1,9 @@
 // ASSESSMENT 4: JavaScript Coding Practical Questions with Jest
 
+const { exportAllDeclaration } = require("@babel/types")
+const { isTypedArray } = require("util/types")
+const { describe, array } = require("yargs")
```

Comment:
This was stopping your tests from running properly.  VS Code occasionally adds this which unfortunately does not help us.